deploy/example_clusters/cr-cluster-training-engine.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: "kubedirector.hpe.com/v1beta1"
+kind: "KubeDirectorCluster"
+metadata: 
+  name: "training-engine"
+spec:
+  app: training-engine
+  roles:
+  - id: RESTServer
+    members: 1
+    resources:
+      limits:
+        cpu: "1"
+        memory: 2Gi
+      requests:
+        cpu: "1"
+        memory: 2Gi
+  - id: LoadBalancer
+    members: 1
+    resources:
+      limits:
+        cpu: "1"
+        memory: 2Gi
+      requests:
+        cpu: "1"
+        memory: 2Gi
+  - id: controller
+    members: 1
+    resources:
+      limits:
+        cpu: "1"
```

Comment:
"controller" will run a notebook process and might run some data-intensive job, let us increase CPU/memory to 2/8Gi